README.md

```diff
@@ -165,150 +162,126 @@ upper right corner.  ![](https://raw.githubusercontent.com/deephaven-examples/de
-6) Use `host=localhost` for the hostname in the examples
+### Deactivate the Virtual Environment
+
+To deactivate the virtual environment:
+```bash
+deactivate
+```
+
+Once the virtual environment is deactivated, `python` and `pip` will use the system's Python and packages.
+[deephaven-ib](https://github.com/deephaven-examples/deephaven-ib) will not be available.
+
+
+# Use deephaven-ib
+
+## Start Python
+
+To start Python with the virtual environment, run:
+```bash
+source ./venv-<versiondetails>/bin/activate
+python
+```
+
+## Start Deephaven
 
-# Authentication
+To start Deephaven, run the following python script.  It can be found at [./examples/run_deephaven.py](./examples/run_deephaven.py).  
+This command will start Deephaven with 4GB of memory and the password `DeephavenRocks!`.
+
+```python
+import os
+from time import sleep
+from deephaven_server import Server
+
+_server = Server(port=10000, jvm_args=['-Xmx4g','-Dauthentication.psk=DeephavenRocks!','-Dstorage.path=' + os.path.expanduser('~/.deephaven')])
+_server.start()
+
+while True:
+    sleep(1)
+```
+
+> :warning: These deephaven server commands **must** be run before importing `deephaven` or `deephaven_ib`.
 
 The documentation and examples here illustrate using Deephaven's [Pre-Shared Key (PSK) authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-psk/)
 with the password `DeephavenRocks!`.  Other types of Deephaven authentication can also work.  
 See the [Deephaven Documentation](https://deephaven.io/core/docs/) for details.
 
-Common ways to authenticate are:
-* [Anonymous Authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-anon/)
-* [Pre-Shared Key (PSK) Authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-psk/)
-* [Username / Password Authentication](https://deephaven.io/core/docs/how-to-guides/authentication/auth-uname-pw/)
+## Launch the Deephaven IDE
 
-# Use deephaven-ib
+Once the Deephaven server is started, you can launch the Deephaven IDE.  
+The Deephaven IDE is a web-based interface for working with Deephaven.
+Once in the IDE, you can run queries, create notebooks, and visualize data.
+You can also run all of the example code below and the more complex examples in [./examples](./examples).
 
-To use [deephaven-ib](https://github.com/deephaven-examples/deephaven-ib), you will need to open the [Deephaven Web IDE](http://localhost:10000/ide/) 
-by navigating to [http://localhost:10000/ide/](http://localhost:10000/ide/) in your web browser.  How you authenticate
-will depend upon how authentication is configured.  In the examples here, you will use the password `DeephavenRocks!`.
+To launch the Deephaven IDE, navigate to [http://localhost:10000/ide/](http://localhost:10000/ide/) in your web browser.
```

Comment:
Probably worth noting that we officially support Chromium-based browsers and Firefox. People who use Safari report all kinds of weird issues.